packages/shared/src/components/EmptyScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import classed from '../lib/classed';
```

Comment:
Thanks for making this a uniform component, will be needing this for other stuff as well 🎉